highfidelity/hifi
# Fix particle uniform buffer issue

Particles were always using 0 as the location of their uniform buffer, which was interfering with the deferred lighting uniform buffers (possibly because there's something wrong with custom pipelines?).  This changes the location used by particles (and polylines, just in case) to avoid the issue.

https://highfidelity.fogbugz.com/f/cases/11289/Texture-rendering-changes-based-on-camera-angle

Test plan:
- Launch the sandbox from this PR so that you have the default content set.  From your starting location, look around (see the gif in the fogbugz case).  The two teleporter entities (the one on the table near you and the one that takes you to a different domain) should continue to render normally.
- The particle effects that make the smoke throughout the area should render correctly.
- Download the Fingerpaint app and draw some polylines.  They should render correctly.
